index.html

```diff
@@ -50,8 +50,7 @@
 
 $('#currentSelection').html(window.apiVersions.current + ' [Current] <span class="caret"></span>');
 
-$('#versionsList').append()
-window.apiVersions.current
+$('#versionsList').append();
```

Comment:
Since nothing is being appended here, you can remove this line

Reply:
👍